firebase_auth_oauth/android/build.gradle

```diff
@@ -41,4 +42,6 @@ android {
 
 dependencies {
     implementation "org.jetbrains.kotlin:kotlin-stdlib-jdk7:$kotlin_version"
+    implementation 'com.google.firebase:firebase-auth'
```

Comment:
The migration guide says 

> Ensure that you're not manually importing any Firebase Android SDK dependencies

We should have access to FirebaseAuth classes from its Flutter plugin which we have as a dependency.

Reply:
Hmm, `firebase_auth` plugin is not exposing this dependency anymore. Makes sense to add it then.